From 0484f67c435dd935d2498338d2d96cd1d095b0fe Mon Sep 17 00:00:00 2001
From: Josh Wilsdon <jwilsdon@joyent.com>
Date: Tue, 9 Apr 2019 10:14:03 -0700
Subject: [PATCH] TRITON-1383 calculate-locality-hints doesn't exist, but is in
 config/test.json which breaks CNAPI's tests Reviewed by: Orlando Vazquez
 <orlando@joyent.com> Approved by: Orlando Vazquez <orlando@joyent.com>

---
 config/test.json | 1 -
 package.json     | 2 +-
 2 files changed, 1 insertion(+), 2 deletions(-)

diff --git a/config/test.json b/config/test.json
index d40db48..62b1df1 100644
--- a/config/test.json
+++ b/config/test.json
@@ -76,7 +76,6 @@
           "hard-filter-min-ram",
           "hard-filter-min-cpu",
           "hard-filter-min-disk",
-          "calculate-locality-hints",
           "hard-filter-locality-hints",
           "hard-filter-owners-servers",
           ["or", "hard-filter-reservoir",
diff --git a/package.json b/package.json
index a638b0e..d8b93e0 100644
--- a/package.json
+++ b/package.json
@@ -1,7 +1,7 @@
 {
   "name": "cnapi",
   "description": "Triton Compute Node API",
-  "version": "1.23.1",
+  "version": "1.23.2",
   "author": "Joyent (joyent.com)",
   "private": true,
   "dependencies": {
-- 
2.21.0

